pkgs/applications/networking/instant-messengers/element/element-web.nix

```diff
@@ -18,73 +18,93 @@ let
     disable_guests = true; # disable automatic guest account registration at matrix.org
     piwik = false; # disable analytics
```

Comment:
piwik is removed for quite some versions https://github.com/vector-im/element-web/commit/6c375748576f4e991f2ed638de119eb1611fa4b8

Reply:
#200073